dashboard/test/models/pd/application/teacher1819_application_test.rb

```diff
@@ -66,7 +66,7 @@ class Teacher1819ApplicationTest < ActiveSupport::TestCase
       teacher_application = build :pd_teacher1819_application, response_scores: {
         free_lunch_percent: '5',
         underrepresented_minority_percent: '5',
-        able_to_attend_single: 'Yes',
+        able_to_attend_single: Teacher1819ApplicationConstants::TEXT_FIELDS[:able_to_attend_single],
```

Comment:
The constants are already [included](https://github.com/code-dot-org/code-dot-org/blob/dc1dcbbcfe5f2c0a96c34b7c56ec7b2a2eff14f9/dashboard/test/models/pd/application/teacher1819_application_test.rb#L6), so you can reference `TEXT_FIELDS` directly.